apps/site/assets/ts/schedule/components/line-diagram/ExpandableBranch.tsx

```diff
@@ -36,8 +36,8 @@ const ExpandableBranch = ({
               style={{ color: `#${color}` }}
             >
               <>
-                {times(branchData[0].stop_data.length - 1, () => (
-                  <div className="m-schedule-diagram__line" />
+                {times(branchData[0].stop_data.length - 1, index => (
+                  <div key={index} className="m-schedule-diagram__line" />
```

Comment:
While normally advised-against, this should be safe here because it's purely static content and will never update after the initial render.